cves/kernel/CVE-2019-9503.yml

```diff
@@ -175,8 +183,12 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    I believe it is possible for an automated tool to discover this vulnerability.
+    The most likely abuse case for this vulnerability results in a denial of service
```

Comment:
The automated tool would need to be a maliciously programmed Wifi dongle  or a fuzzer of some sort no?

Reply:
Updated.